tests/Microsoft.Bot.Builder.Azure.Tests/CosmosDbStorageTests.cs

```diff
@@ -407,6 +411,76 @@ public async Task WaterfallCosmos()
             }
         }
 
+        [TestMethod]
+        public async Task DeleteAsyncFromSingleCollection()
+        {
+            string documentId = "UtteranceLog-001";
+            string[] keys = { documentId };
+            var storage = new CosmosDbStorage(CreateCosmosDbStorageOptions());
+            var item = new StoreItem { MessageList = new string[] { "hi", "how are u" }, City = "Contoso" };
+            var changes = new Dictionary<string, object>();
+            {
+                changes.Add(documentId, item);
+            }
+
+            await storage.WriteAsync(changes, CancellationToken.None);
+
+            var result = await Task.WhenAny(storage.DeleteAsync(keys, CancellationToken.None)).ConfigureAwait(false);
+            Assert.IsTrue(result.IsCompletedSuccessfully);
+        }
+
+        [TestMethod]
+        public async Task DeleteAsyncFromPartitionedCollection()
+        {
+            /// The WriteAsync method receive a object as a parameter then encapsulate it in a object named "document"
+            /// The partitionKeyPath must have the "document" value to properly route the values as partitionKey
+            /// <seealso cref="WriteAsync(IDictionary{string, object}, CancellationToken)"/>
+            string partitionKeyPath = "document/city";
```

Comment:
The `partitionKeyPath`, `documentId` & `keys` variables are also repeated along the three new test methods. Can we take a similar approach as is being done [here](https://github.com/southworkscom/botbuilder-dotnet/blob/ed62477a2fee0479e688dd0ef4dfd923ba760a31/tests/Microsoft.Bot.Builder.Azure.Tests/CosmosDbStorageTests.cs#L28), I.E, create `private` `const` variables for not being constantly repeating these parameters???

Reply:
I will promote the local variables to be global.